### ISSUE TITLE
Is it possible to add an icon on the Activitusbar that invokes the lowermost icon on the activity bar?

### ISSUE THREAD

**Issue by septsea** [Gruntfuggly/activitusbar#50]
Hi! Your extension is so nice.

I wonder whether or not it is possible to add an icon on the Activitusbar that invokes the lowermost icon on the activity bar.

![image](https://user-images.githubusercontent.com/57337516/137574654-9da732a5-2355-4a59-b336-5509003509dc.png)

**Comment by Gruntfuggly** [comment]
No sorry, there's no API open to extensions for that sort of thing.

It might be possible to create a custom menu with similar commands (not all would be available), but it would open at the top of the screen, so you might as well use the normal menus.

It might be easier to add buttons for the menu options you want the most?

**Comment by Gruntfuggly** [comment]
Yes, sounds like shortcuts might be your best option. The only other thing you could do is add a button activitusbar to open the activity bar (and close it again). You'd still need two clicks though.

Are you OK if I close this now?
